### DISCUSSION TITLE
Is SC 1.16.1 Supported?

### DISCUSSION THREAD

**Discussion by TheJoe49** [Markemp/Cryengine-Converter#121] [Q&A] [answered]
I am trying to convert some ships from star citizen and I keep getting models that are missing a bunch of parts or that are smooth and missing detail. I didn't want to open a ticket cause I totally realize it could just be me not knowing what I'm doing. Any help would be appreciated thanks
![Screenshot_2](https://user-images.githubusercontent.com/16162727/151486344-a9c6e71f-be78-4aec-a803-df0109c1495c.png)
!

**Comment by Baconator650** [comment]
That looks pretty normal for a importing just Body of the ship (thrusters, weapons, interior are separate models). Are you using just CGF-Converter or StarFab which uses the CGF-Converter as its model converter?

**Reply by TheJoe49** [reply]
Is it normal for the front of the ship to be low res like that? And I'm using the unp4k suite and then CGF-Converter. Also is it normal for the the thrusters and weapons to import to the origin instead of where they would be on the ship?

**Reply by Baconator650** [reply]
Yes it completely normal as Markemp has already pointed out. It has to do with the shielding effects and the particle collision box for the quantum travel. It is normal for thrusters in weapons to be import in the center as they independent objects that get attach to hardpoints. If you go the discussion area and go to "Star Citizen Small Ship Conversion (Pre-SCDV)" and watch my video series there if will be go through the pre-SCDV/StarFab process.

**Reply by TheJoe49** [reply]
Thanks too!

**Comment by Markemp** [comment, marked as answer]
The ships have a shields layer that is low resolution that covers the physical geometry.  In Blender you can select all vertices by material, select the shield material, and delete the vertices.  The physical ship should be visible then.

You can also assign a transparent material to it, or I think have it hide a material as well.  Probably a few ways to get around that.

**Reply by TheJoe49** [reply]
Oh Awesome, Thanks For the Help!
